modules/integration-browser/src/integration.encrypt.test.ts

```diff
@@ -60,7 +60,8 @@ function aTest(testName: string, decryptOracle: string) {
       needs(response.ok, `Failed to decrypt: ${toUtf8(body)}`)
       expect(plainText).toEqual(new Uint8Array(body))
     } catch (e) {
-      if (!notSupportedMessages.includes(e.message)) throw e
+      if (e instanceof Error && !notSupportedMessages.includes(e.message))
+        throw e
```

Comment:
I think that this logic is not perfect. If something is not supported it is not an error. However in every other case it _is_ an error.

So I think that `needs` may be a better solution.

e.g.

```
needs(e instanceof Error && notSupportedMessages.includes(e.message))
```

Reply:
I struggled with this for most of yesterday. Did not make any progress...